### DISCUSSION TITLE
Allow custom view component directory

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#414] [Feature requests]
The custom field generator generates several view components that are used to render the field. It would be nice if the generator could make use of a configuration option that allows the directory of these generated components to change. Currently, it places the components in `app/components`, but in my apps, I put components in different directories, such as `app/view_components` or `app/frontend/components`.

**Comment by adrianthedev** [comment]
I see what you mean. Yeah, that could be a nice addition.

**Reply by adrianthedev** [reply]
I'll add it as an issue so someone picks it up.

**Reply by adrianthedev** [reply]
https://github.com/avo-hq/avo/pull/423
